src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -61,8 +61,8 @@ for (const section of sidebarSections) {
 <script>
 	window.addEventListener('DOMContentLoaded', () => {
 		var target = document.querySelector('[aria-current="page"]');
-		if (target && target.offsetTop > window.innerHeight - 100) {
-			document.querySelector('.nav-groups').scrollTop = target.offsetTop;
+		if (target && target.offsetTop > window.innerHeight - 120) {
+			document.querySelector('.nav-groups').scrollTop = target.scrollTop + target.offsetTop - 50;
```

Comment:
I’d love a comment here explaining what `120` and `50` mean for future Astro docs archaeologists 😅 

```suggestion
			document.querySelector('.nav-groups').scrollTop = target.scrollTop + target.offsetTop - 50;
```

Reply:
As an example, is 50 the height of one menu item? 120 some padding the page has? That kind of thing.